src/styles/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  color: $main-color;
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding: 16px;
+  display: flex;
+  flex-direction: column;
+
+  @include transition(all);
+
+  &__image {
+    margin: 16px 3px 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__description {
+    margin-bottom: 16px;
+    display: flex;
+    flex-direction: column;
+    gap: 4px;
+
+    &-name {
+      @include regular-text;
+
+      font-weight: 500;
+    }
+
+    &-code {
+      @include small-text;
+
+      color: $secondary-color;
+    }
+  }
+
+  &__rate {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 24px;
+
+    &-reviews {
+      @include small-text;
```

Comment:
The mixin `small-text` should be defined in your mixins file. Ensure that it is correctly defined and imported if not already.